src/Microsoft.AspNet.Razor/Parser/TokenizerBackedParser.cs

```diff
@@ -75,6 +76,39 @@ protected void Initialize(SpanBuilder span)
             }
         }
 
+        protected TSymbol Lookahead(int count = 1)
+        {
+            if (count == 0)
+            {
+                throw new ArgumentOutOfRangeException(nameof(count));
```

Comment:
`throw` if `count < 0` (a case that's missing).  return `CurrentSymbol` when `count == 0`